packages/manager/modules/pci/src/projects/project/analytics-data-platform/deploy/storage/storage.html

```diff
@@ -107,10 +101,7 @@ <h4><strong data-translate="analytics_data_platform_deploy_storage_edge_node_tit
                 <dt class="storage-list__term"><strong><span data-translate="analytics_data_platform_deploy_storage_total_effective_storage"></span> :</strong></dt>
                 <dd class="storage-list__definition">
                     <strong>
-                        <span>
-                            {{ $ctrl.data.edgeNodeStorage * $ctrl.nodesConfig.edge.count }}
-                        </span>
-                        <span data-translate="analytics_data_platform_common_unit_gb"></span>
+                        <span data-ng-bind="$ctrl.getTotalEffectiveStorage()"></span>
```

Comment:
The `ngBind` directive could be applied directly on the `<strong>` parent element, right?

Reply:
Yes... This has been fixed. Thanks :)